RadarSDK/RadarLocationManager.m

```diff
@@ -240,10 +240,19 @@ - (void)requestLocation {
 }
 
 - (void)updateTracking {
-    [self updateTracking:nil];
+    [self updateTracking:nil fromInitialize:NO];
+}
+
+- (void)updateTrackingFromInitialize:(BOOL)fromInitialize {
+    [self updateTracking:nil fromInitialize:fromInitialize];
 }
 
 - (void)updateTracking:(CLLocation *)location {
+    [self updateTracking:location fromInitialize:NO];
+}
+
+- (void)updateTracking:(CLLocation *)location
+        fromInitialize:(BOOL)fromInitialize {
```

Comment:
again here - what's best practice? Could it be `withIsFromInitiailize`?

Reply:
the second parameter and beyond in objective-c tends to be redundant / wordy due to the quirky way it organizes them - https://stackoverflow.com/questions/3911593/how-to-call-a-method-from-another-method-in-objective-c